src/includes/blogpost.njk

```diff
@@ -1,4 +1,9 @@
 <div class="blogpost">
+  <div class="blogpost-breadcrumb">
+    <p>
```

Comment:
I'm wondering if a paragraph makes sense here?

Reply:
not anymore